### PR TITLE
Kelp gain integration

### PR DESCRIPTION
**Pull request by AdvayAgrawal**
# Checklist

- [ ] App update process has been followed 
- [ ] Target branch is `develop` 
- [ ] Application version has been bumped

### CODE REVIEW

File: src/plugin.h [hunk 1]
```diff
@@ -66,6 +70,7 @@ typedef struct context_s {
     uint8_t account_addr[ADDRESS_LENGTH];
     uint8_t token_addr[ADDRESS_LENGTH];
     char ticker[MAX_TICKER_LEN];
+    char receive_ticker[MAX_TICKER_LEN];
```
Comment (0xWinDefi) [inline src/plugin.h]: remove this?

File: src/handle_init_contract.c [hunk 2]
```diff
@@ -60,6 +62,16 @@ void handle_init_contract(ethPluginInitContract_t *msg) {
...
             break;
 
+        case GAIN_DEPOSIT_RSETH:
+            context->next_param = STAKE_AMOUNT;
+            strlcpy(context->ticker, "RSETH", sizeof(context->ticker));
+            break;
+
+        case GAIN_WITHDRAW:
+            context->next_param = UNSTAKE_AMOUNT;
+            strlcpy(context->ticker, "RSETH", sizeof(context->ticker));
```
Comment (0xWinDefi) [inline src/handle_init_contract.c]: i think not required ?

File: src/handle_query_contract_ui.c [hunk 3]
```diff
@@ -70,6 +70,57 @@ static bool handle_kelp_initiate_withdraw(ethQueryContractUI_t *msg, context_t *
...
+            break;
+
+        case 1:
+            strlcpy(msg->title, "Receiver", msg->titleLength);
+            ret = set_account_addr_ui(msg, context);
+            break;
+
+        case 2:
+            strlcpy(msg->title, "Asset Expected", msg->titleLength);
+            strlcpy(msg->msg, context->ticker, msg->msgLength);
```
Comment (0xWinDefi) [inline src/handle_query_contract_ui.c]: directly use "RSETH" here ?

File: src/plugin.h (hunk 1 above)
Comment (AdvayAgrawal) [reply src/plugin.h]: done

**Pull request merged**